nucliadb_utils/src/nucliadb_utils/audit/audit.py

```diff
@@ -61,28 +57,29 @@ async def initialize(self):
     async def finalize(self):
         pass
 
-    async def visited(self, kbid: str, uuid: str, user: str, origin: str):
-        raise NotImplementedError
-
-    async def search(
+    async def visited(
         self,
         kbid: str,
+        uuid: str,
         user: str,
-        client: int,
         origin: str,
-        search: SearchRequest,
-        timeit: float,
-        resources: int,
+        send: bool = False,
```

Comment:
This boolean deciding whether to call or not another public method, mixed with all the audit parameters, feels kind of weird.
As the only user is ingest, how do you feel calling `report` and `send` or a "`report_and_send`" method?

Reply:
This made me realize that they were some audit functions already deprecated and some other that only used the inner kbusage utility, i decided to split the audit and report into completely independent utilities and use them explicitly where needed. This way the only "send=True" needed was in the `report`.